### PR TITLE
Add com.github.optyfr.JRomManager

### PR DESCRIPTION
**Pull request by FakeShemp**
https://github.com/optyfr/JRomManager

Not built from source because flatpak-builder-tools has no support for Gradle/Maven repos yet.

### CODE REVIEW

**Comment by FakeShemp** [comment]
bot, build com.github.optyfr.JRomManager

**Comment by FakeShemp** [comment]
bot, build com.github.optyfr.JRomManager

File: com.github.optyfr.JRomManager.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        for s in {16,24,32,48,72,96,128,192,256,512}; do
+        rsvg-convert "${icon_in}" -w "${s}" -h "${s}" -a -f png -o "${icon_out}";
+        install -p -Dm644 "${icon_out}" -t "${FLATPAK_DEST}/share/icons/hicolor/${s}x${s}/apps/";
+        done;
+      - install -Dm644 JRomManager.desktop "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - desktop-file-edit --set-key=Exec --set-value=JRomManager-multi.sh "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - desktop-file-edit --set-key=Icon --set-value=${FLATPAK_ID} "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm644 appdata.xml "${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml"
+    sources:
+      - type: file
```
Comment (barthalion) [inline com.github.optyfr.JRomManager.yaml]: Include also source tarball so f-b exports it into Sources extension.

File: com.github.optyfr.JRomManager.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: JRomManager
+    buildsystem: simple
+    build-commands:
+      - unzip -d ${FLATPAK_DEST}/bin JRomManager-2.0.3.zip
+    post-install:
```
Comment (barthalion) [inline com.github.optyfr.JRomManager.yaml]: Everything in post-install should land in build-commands for clarity.

**Comment by barthalion** [comment]
Have you contacted upstream developer about your submission?

**Comment by nedrichards** [comment]
@FakeShemp yeah, we need to see a communication with upstream before we add community maintained projects these days.

**Comment by FakeShemp** [comment]
bot, build com.github.optyfr.JRomManager

**Comment by FakeShemp** [comment]
bot, build com.github.optyfr.JRomManager

**Comment by optyfr** [comment]
> Have you contacted upstream developer about your submission?

Yes, I've been contacted :)
Glad to be in flathub

**Comment by nedrichards** [comment]
Great! Then all we're looking for is addressing this https://github.com/flathub/flathub/pull/895#pullrequestreview-212552801 and we can add it ASAP.

**Comment by nedrichards** [comment]
Oh, wait, that happened. Hurrah!

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.optyfr.JRomManager

**Pull request closed without merging**